docs/quickstart.md

```diff
@@ -0,0 +1,72 @@
+# Quick Start:
+
+The most basic Spiel presentation looks like this:
```

Comment:
I *might* prefer if we just linked here from the `README.md` to de-duplicate this chunk. Maybe the `README.md` should be very code-example-y? Just a chunk of sample code and the resulting output with a line or two of text, then link here?

Reply:
I personally tend to like it when a README gives me an overview of what to do without having to click off into another page. It is a little clumsy to have the same content copy/pasted both places, so I'll let you make the final call on what the README should have vs. link out to!

Reply:
Fair enough! I'll think about this some more and make the final edits on this point before merging.